terraform/Using_Make.md

```diff
@@ -119,16 +120,17 @@ public_vlan_number  = "2979230"
 
 This is the recommended option to follow when the Terraform code is working correctly or if you are doing changes in any of the the `cp4*/workspace.tmpl.json` files.
 
-Make sure everything is working provisioning the Cloud Pak using Schematics before release it.
+Make sure everything that provisioning the Cloud Pak using Schematics works before releasing.
 
-9. Execute this command to create the Schematics workspace:
+1. Execute this command to create the Schematics workspace:
 
    ```bash
    make with-schematics
    ```
 
-10. Go to the displayed link to edit or validate the variables in the workspace. By default the cluster is created on `us-south` region and datacenter `dal10`. If you would like to change any of these parameters, edit the variables at the workspace settings.
-    Also, until the permissions issue is not solved you need to provide the VLANs. Execute the command `ibmcloud ks vlan ls --zone {datacenter}`, get a private and public VLAN, and write them down in the variables at the workspace settings. In this example, you can select the VLANs `2979232` (as private) and `2979230` (as public):
+2. Go to the displayed link to edit or validate the variables in the workspace. By default the cluster is created on `us-south` region and datacenter `dal10`. If you would like to change any of these parameters, edit the variables at the workspace settings.
+    
+3. NOTE: Until the permissions issue is solved the VLANs need to be provided. Execute the command `ibmcloud ks vlan ls --zone {datacenter}`, get a private and public VLAN, and write them down in the variables at the workspace settings. In this example, you can select the VLANs `2979232` (as private) and `2979230` (as public):
```

Comment:
The other references to VLAN are a paragraph but this one makes sense to be a list item, therefore I'm removing the `NOTE:` because does not looks good. 

Please, feel free to modify this item to make enfasis on it but I'd suggest not to use `NOTE:` :)